### DISCUSSION TITLE
How to get html load duration statistics in performance tab？

### DISCUSSION THREAD

**Discussion by reuwi** [getsentry/support#26] [Support]
In performance tab, web vitals including FIP,FCP etc was shown, and resources load time could be viewd in events list page, but it seems the html load time is shown nowhere yet🤔

**Comment by souredoutlook** [comment]
Hey @reuwi thanks for your patience while we work through this discussion backlog - can you send me a link to the event details page where you're seeing resources load time that you would like displayed on the Web Vitals tab, just so I can verify we're on the same page.

For example, it's possible to swap out the default visualizations for worst-performing web vitals for other metrics, p.75 HTTP, Browser and Resource operations. Screenshot below:

<img width="593" alt="screenshot showing the results of selecting the meatball menu icon on a visualization from the web vitals tab" src="https://user-images.githubusercontent.com/76002357/187686471-4f07c134-ef7b-4cf5-ba37-7ac96141c187.png">

Is this what you're looking for?
